nursery/get-tcp-endpoints.yml

```diff
@@ -0,0 +1,10 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: get TCP endpoints
```

Comment:
what do you mean by "endpoints"?

is this enumerating TCP connections? sessions? listeners?

Reply:
`GetExtendedTcpTable` can return connections, listeners, or both depending on which [TCP_TABLE_CLASS](https://docs.microsoft.com/en-us/windows/win32/api/iprtrmib/ne-iprtrmib-tcp_table_class) is specified. Maybe the name could be changed to `list TCP connections and listeners`?